### ISSUE TITLE
Issue with "HTMLFormElement: formdata event": The example code is bonkers

### ISSUE THREAD

**Issue by salmatron** [mdn/content#3886]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/formdata_event

#### What information was incorrect, unhelpful, or incomplete?
The 'formdata' event is being used to fire an ajax request.
```
// submit the data via XHR
var request = new XMLHttpRequest();
request.open("POST", "/formHandler");
request.send(data);
```
#### Specific section or headline?
Code under "Examples"

#### What did you expect to see?
`formdata` event is to be used as a hook by custom implementations of form elements to modify the form output (by mutating `e.formData`) before it's submitted/serialized, it is NOT a substitute of the `submit` event. I shudder to think that if the example code made into real code-bases then it will prevent adoption of valid usages of this event.

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/htmlformelement/formdata_event`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/formdata_event
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/htmlformelement/formdata_event/index.html
* Last commit: https://github.com/mdn/content/commit/babc0473793bbe3cfa66cd96fae7f84348be373b
* Document last modified: 2021-02-08T10:05:46.000Z

</details>

**Comment by Salmatron** [comment]
The person writing that example might have been mislead by the description here (which is also incorrect) https://developer.mozilla.org/en-US/docs/Web/API/FormDataEvent

> [...] allows a FormData object to be quickly obtained in response to a formdata event firing, rather than needing to put it together yourself when you wish to submit form data via a method like XMLHttpRequest (see Using FormData objects).

**Comment by Josh-Cena** [comment]
This has been fixed via #8973
